src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac Product Image"
+          class="card__image"
```

Comment:
The class name 'card__image' is used for both the div and img elements. This can lead to confusion and unexpected styles being applied. Consider using more descriptive and unique class names for each element. For example, 'card__image-container' for the div and 'card__image' for the img.